### PR TITLE
fix: fix broken animation SWA flow Android

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [] 

### Description

Resolves [[Android] Submit Artwork → screen grayed out on review but responsive](https://www.notion.so/artsy/Android-Submit-Artwork-screen-grayed-out-on-review-but-responsive-130cab0764a080b8b027fabecf354f41)

Set `transitionSpec` to undefined on the Stack.Navigator's screenOptions->cardStyle for Android
`transitionSpec` was causing unnecessary delay when transitioning between the screens making the transition between freeze.  We still use the `cardStyleInterpolator` setting that adds necessary animation for both Android and iOS.
For a simple use case, just setting cardStyleInterpolator can often be enough to give the desired visual effect without modifying the transition’s timing.

| Before | After |
| --- | --- |
| [Video](https://www.notion.so/artsy/Android-Submit-Artwork-screen-grayed-out-on-review-but-responsive-130cab0764a080b8b027fabecf354f41) | https://github.com/user-attachments/assets/88fbfc61-dd97-4370-8412-552bc523089d |

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- 

#### iOS user-facing changes

-

#### Android user-facing changes

- Set transitionSpec to undefined on the Stack.Navigator's screenOptions->cardStyle for android - daria

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/Scenes/SellWithArtsy/ArtworkForm/SubmitArtworkForm.tsx [hunk 1]
```diff
@@ -181,20 +181,23 @@ const SubmitArtworkFormContent: React.FC<SubmitArtworkProps> = ({
...
+                        },
+                      },
+                      close: {
+                        animation: "timing",
+                        config: {
+                          duration: 300,
+                        },
+                      },
+                    }
+                  : undefined,
```
Comment (gkartalis) [inline src/app/Scenes/SellWithArtsy/ArtworkForm/SubmitArtworkForm.tsx:200]: 🤔 good catch

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Android user-facing changes (Set transitionSpec to undefined on the Stack.Navigator's screenOptions->cardStyle for android - daria - dariakoko)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 181ccc3c6dddc48720d3c7155282bc45d5363601
</p>

**Review by gkartalis — COMMENTED** [review]
looks great for my side, @MrSltun could you maybe try to reproduce this?

**Comment by olerichter00** [comment]
Thanks for the fix 🙏

I like the idea of removing the spec for iOS as well if there is no noticeable difference. But it would be good to find out why we have added it in the first place 🤔

**Review by MrSltun — APPROVED** [review]
Just tested it locally on my device and it looks great! Thank you Daria! 🌟🌟🌟

**Comment by dariakoko** [comment]
Agree, @olerichter00 
If I'm not mistaken, @MounirDhahri was the one working on the first steps of the new SWA flow?
Let's merge it now to unblock the release and follow up on iOS later

**Comment by MrSltun** [comment]
@dariakoko we might need to rebase from `main` due to #11050 being merged

**Pull request merged**